### ISSUE TITLE
Issues with Demo

### ISSUE THREAD

**Issue by dariusjulian** [RenderHeads/UnityPlugin-AVProMovieCapture#273]
Hello,
I'm evaluating the demo to see if it meets my needs for a project but I'm running into issues.
I'm having trouble trying to run the Webcam capture demo.
1. The webcam frame rate is around 5fps which is extremely slow.
2. When I hit play in the editor the only way I am able to start recording is by assigning a toggle key and using that.  Using the GUI or inspector buttons to start recording doesn't do anything.
3. When I do start recording, an error is immediately thrown saying "[AVProMovieCapture] No texture set to capture".
4. After receiving the above error if I unpause the app it will begin recording as expected.

I'm running Unity 2021.3.9f1 on an i7-9700 3.6GHz and an Nvidia GeForce RTX 2060 Super.
Any assistance would be appreciated.

Regards,

Darius

**Comment by Chris-RH** [comment]
Hi @dariusjulian, 

I've just retested a new project with the trial version 5.1.4, with Unity 2021.3.21, Demo02_WebcamCapture and I'm not seeing these issues.

What webcam are you using? Are the drivers up to date? Is the room well lit?
Are you using Windows 10?11?
Linear or Gamma?
D3D11 or 12?
Any other information that you could provide?

**Comment by dariusjulian** [comment]
Hi Chris,

Thank you for the timely feedback.
I'm using Windows 10 Pro 64bit.
I'm using an HD Pro Webcam C920
I'm using Forward rendering and Gamma Colorspace and had auto graphics-api checked.

Just now I unchecked it and selected D3D11 and tried to run the demo again.  No luck and now it doesn't record when I unpause the demo after the first error mentioned in my previous post.

Originally I simply created a new project.  Imported the trial package and opened the webcam demo scene and hit play.  I've been tinkering a little since then but haven't made any changes to anything other than choosing the D3D11 api.

Regards,

Darius

**Comment by Chris-RH** [comment]
Ahh, are you using URP or HDRP? and which version?

**Comment by dariusjulian** [comment]
I'm actually not using either one.  I'm using the built in renderer.  That said I did finally get it working perfectly for my needs.  I created a new scene, added the webcamcapture prefab and wrote my own controller code.  I created a webcam texture and threw it on an image and used that as the source for the prefab.  It took a little fiddling but I got it recording at 30fps and it was lightning fast as far as creating and saving the recordings.  I was even able to green screen the webcamtexture and still record the un-keyed stream; which was my ultimate goal.  Thank you for your help.  I'll be picking up this asset soon!

Regards,

Darius
